extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -148,6 +148,7 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 					{
 						title: constants.enterConnectionString,
 						ignoreFocusOut: true,
+						value: 'Server=localhost;Initial Catalog={db_name};Persist Security Info=False;User ID=sa;Password={your_password}',
```

Comment:
Would suggest ordering it like this 

`Server=localhost;Initial Catalog={db_name};User ID=sa;Password={your_password};Persist Security Info=False`

So that both the db_name and password are fully visible from the start